ConsoleApp1/Program.cs

```diff
@@ -7,6 +7,7 @@ class Program
         static void Main(string[] args)
         {
             Console.WriteLine("Hello World!");
+            Console.WriteLine("Enter your name:");
```

Comment:
This is cool, i never thought of this before. 
but can you add something more here.

Reply:
Sure!

Reply:
Thanks.